.travis.yml

```diff
@@ -19,6 +19,10 @@ addons:
     packages:
       - postgresql-12
       - postgresql-client-12
+env:
```

Comment:
I already have an `env:` section (see line 54), is that going to be a problem if there are 2 entries with the same name in the config file ? Or should this be moved to the existing section below ?

Reply:
@dregad I have updated the env var config to test it out.